Client/GameUI.cpp

```diff
@@ -2926,7 +2926,9 @@ UI_SetHP(int current, int max)
 			ExecuteHelpEvent( HELP_EVENT_USE_POTION );

```

Comment:
It's not really useful, because `ExecuteHelpEvent` seems to be an empty function.